SocialFilter/sections/css_extended.txt

```diff
@@ -17,6 +17,7 @@ m.hkgolden.com#?#div.MuiGrid-container > div.MuiGrid-root:has(> button[aria-labe
 livejournal.com#?#.social-panel > span:has(> button[class^="socialShareBtn-"] > span:not([title="Репост в ЖЖ"]):not([title="Embed"]):not([title="Копировать URL"]))
 jornaldocarro.estadao.com.br#?#.row > div.col:has(> div.jc-post-social-media-share)
 press.jal.co.jp#?#.l-contents__body > div.entry-body + div.l-row div.__elem:contains(SNS):upward(3)
+houjin.info#?#.widget_nav_menu > h5.widget-title:contains(SNSでシェアする):upward(1)
```

Comment:
It's better to use `:has` if possible, because it has a native support in CSS.
For example:
`houjin.info#?#.span4 > div.widget:has(> div.menu-menu-1-container > ul.circle_group)`

Reply:
I fixed it by myself: https://github.com/AdguardTeam/AdguardFilters/commit/5984af934a7271e4d8bb2816a6064640564f8081

Reply:
Thank you. And Sorry for my late